File: PL027.cpp
#include <iostream>
#include <bits/stdc++.h>
using namespace std;

string oddOrEven(int n)
{
    if (n % 2 == 0)
    {
        return "Even";
    }

    return "Odd";
}

int main()
{
    int n;
    cout << "Enter the number : " << endl;
    cin >> n;
    cout << "The number is : " << oddOrEven(n) << endl;
    return 0;
}

File: PL027.java
import java.util.Scanner;

public class PL027 {
    public static void main(String[] args) {
        int n;
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the number : ");
        n = scanner.nextInt();
        scanner.close();
        System.out.println("The number is : " + oddOrEven(n));
    }

    public static String oddOrEven(int n) {
        if (n % 2 == 0) {
            return "Even";
        }

        return "Odd";
    }
}
